docroot/modules/custom/my_assets/css/custom-table.css

```diff
@@ -0,0 +1,6 @@
+td:nth-child(even) {
```

Comment:
directory name should be `assets` rather `my-assets`

Reply:
It is module name, It can be anything, and I kept it this way to avoid any conflict with existing directory.
It is not assets directory but module directory. I can create assets directory inside it if mandatory.